sdk/samples/iot/pnp/pnp_protocol.c

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "iot_sample_common.h"
+#include <iot_sample_common.h>
```

Comment:
Good eye.
I was reading up on the difference, and my understanding is that "" means to check current directory (or sub directories), while <> means to check via a directory list determined by the compiler. Does that sound accurate?

Reply:
yea IIRC it has to do with precedence of lookup. `""` is local directory first while I believe `<>` doesn't look up locally.